### DISCUSSION TITLE
各位大佬，为什么我的 waline 不显示头像啊

### DISCUSSION THREAD

**Discussion by wgx-dh** [YunYouJun/hexo-theme-starry#294] [Q&A 疑难杂症] [answered]
![image](https://user-images.githubusercontent.com/63238251/168421708-888e5c78-9ae9-4b79-b8fd-525c9ca0c134.png)
刚更新的 1.9.1 主题
稍微动了一下侧边栏的代码
大部分 cdn 从 jsd 换成了 eleme 
和waline 有关的东西都没动  就是不显示头像
控制台无报错
地址：[https://cuffs.cf/comments/](https://cuffs.cf/comments/)

**Comment by YunYouJun** [comment, marked as answer]
我认为这与你 Waline 的配置有关。请仔细检查配置或 Waline 相关的环境变量。

v1.9.1 版本，无法复现。https://www.yunyoujun.cn/about/

**Comment by wgx-dh** [comment]
就是去检查 部署在vercel服务端的环境变量吗

**Reply by YunYouJun** [reply]
嗯。
